### PR TITLE
Arm/master/atf v2.0.1

### PR DESCRIPTION
**Pull request by marex**
Sync with downstream ATF release 2.0.1 , fix for ULCB board, add myself as a maintainer to help Jorge.

### CODE REVIEW

**Comment by davidcunado-arm** [comment]
@marex 
As per contributing guide, changing PR to target the integration branch.

File: maintainers.rst [hunk 1]
```diff
@@ -186,6 +186,8 @@ Raspberry Pi 3 platform port
 
 Renesas rcar-gen3 platform port
 -------------------------------
+:M: Marek Vasut <marek.vasut@gmail.com>
+:G: `marex`_
```
Comment (antonio-nino-diaz-arm) [inline maintainers.rst]: You need to add the link to the bottom of the document.

File: drivers/staging/renesas/rcar/qos/M3/qos_init_m3_v30.c [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+		io_write_64(QOSWT_BE_WTQOS_BANK0 + i*8,
+				qoswt_be[i]);
+		io_write_64(QOSWT_BE_WTQOS_BANK1 + i*8,
+				qoswt_be[i]);
+	}
+#endif /* RCAR_REWT_TRAINING != RCAR_REWT_TRAINING_DISABLE */
+	}
+
+	/* AXI-IF arbitration setting */
+	//delete
```
Comment (antonio-nino-diaz-arm) [inline drivers/staging/renesas/rcar/qos/M3/qos_init_m3_v30.c]: This comment could be more descriptive.

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

**Comment by marex** [comment]
On 3/4/19 10:56 AM, Antonio Niño Díaz wrote:
> *@antonio-nino-diaz-arm* commented on this pull request.
> 
> ------------------------------------------------------------------------
> 
> In maintainers.rst
> <https://github.com/ARM-software/arm-trusted-firmware/pull/1854#discussion_r261987869>:
> 
>> @@ -186,6 +186,8 @@ Raspberry Pi 3 platform port
>  
>  Renesas rcar-gen3 platform port
>  -------------------------------
> +:M: Marek Vasut <marek.vasut@gmail.com>
> +:G: `marex`_
> 
> You need to add the link to the bottom of the document.

Link to what ? You mean keep the list sorted, alphabetically ?

> ------------------------------------------------------------------------
> 
> In drivers/staging/renesas/rcar/qos/M3/qos_init_m3_v30.c
> <https://github.com/ARM-software/arm-trusted-firmware/pull/1854#discussion_r261988101>:
> 
>> +		io_write_64(QOSWT_FIX_WTQOS_BANK0 + i*8,
> +				qoswt_fix[i]);
> +		io_write_64(QOSWT_FIX_WTQOS_BANK1 + i*8,
> +				qoswt_fix[i]);
> +	}
> +	for (i = 0U; i < ARRAY_SIZE(qoswt_be); i++) {
> +		io_write_64(QOSWT_BE_WTQOS_BANK0 + i*8,
> +				qoswt_be[i]);
> +		io_write_64(QOSWT_BE_WTQOS_BANK1 + i*8,
> +				qoswt_be[i]);
> +	}
> +#endif /* RCAR_REWT_TRAINING != RCAR_REWT_TRAINING_DISABLE */
> +	}
> +
> +	/* AXI-IF arbitration setting */
> +	//delete
> 
> This comment could be more descriptive.

It should not be there at all, dropped.

Rebased on arm/integration and updated the PR.

-- 
Best regards,
Marek Vasut

**Comment by antonio-nino-diaz-arm** [comment]
> Link to what ? You mean keep the list sorted, alphabetically ?

The name between quotes is not a valid link, that's just a reference to a link. You need to go to the bottom of the document and add it.

**Comment by marex** [comment]
On 3/4/19 1:12 PM, Antonio Niño Díaz wrote:
>     Link to what ? You mean keep the list sorted, alphabetically ?
> 
> The name between quotes is not a valid link, that's just a reference to
> a link. You need to go to the bottom of the document and add it.

Oh, OK, let's try it again, pushed.

-- 
Best regards,
Marek Vasut

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Comment by antonio-nino-diaz-arm** [comment]
The CI failures are checkpatch issues in the ``drivers/staging`` folder, we can ignore them.

**Pull request merged**